package test.exceptions;

import org.junit.Before;
import org.junit.Test;

import static org.junit.Assert.*;

public class UserNameOrPasswordAreIncorrectTest {

    @Before
    public void setUp() throws Exception {
    }

    @Test
    public void getUsername() {
    }

    @Test
    public void getPassword() {
    }
}